lib/routes/validators.js

```diff
@@ -151,6 +151,31 @@ module.exports.url = function url(options) {
   return validator
 }
 
+module.exports.pushCallbackUrl = function pushUrl(options) {
+  const validator = isA.string().uri(options)
+  validator._tests.push(
+    {
+      func: (value, state, options) => {
+        if (value !== undefined && value !== null) {
+          let normalizedValue = value;
+          // Fx Desktop registers https push urls with a :443 which causes `isValidUrl`
+          // to fail because the :443 is expected to be normalized away.
+          if (/^https:\/\//.test(value) && /:443($|\/[^\d])/.test(value)) {
```

Comment:
Possible to combine this? Maybe something like this?

```javascript
/^(https:\/\/)[a-zA-Z0-9.-]*(:443)($|\/[^\d])/.test(value)
```

(Although now I see that might be harder to read)